CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+# 1.89.0
+* Enhancement - Bump google.golang.org/grpc from 1.64.0 to 1.64.1 in /ecs-agent/daemonimages/csidrive [#4361](https://github.com/aws/amazon-ecs-agent/pull/4361)
```

Comment:
super nit (non-blocking): typo in `csidrive` (missing `r`)

Reply:
Thanks for catching that.